test/foundry/modules/royalty/IpRoyaltyVault.t.sol

```diff
@@ -11,6 +11,7 @@ import { BaseTest } from "../../utils/BaseTest.t.sol";
 contract TestIpRoyaltyVault is BaseTest {
     event RoyaltyTokensCollected(address ancestorIpId, uint256 royaltyTokensCollected);
     event SnapshotCompleted(uint256 snapshotId, uint256 timestamp, uint32 unclaimedTokens);
+    event RevenueTokensClaimed(address claimer, address token, uint256 amount);
```

Comment:
For tests, you can also import the interface and use the event, ie. `IIpRoyaltyVault.RevenueTokensClaimed`

Reply:
https://github.com/storyprotocol/protocol-core-v1/pull/32/commits/332ac102c9ff545656249d778cf579ceac823c45